src/Events/HttpEvent.php

```diff
@@ -4,16 +4,8 @@
 
 namespace Algatux\InfluxDbBundle\Events;
 
-use Algatux\InfluxDbBundle\Model\PointsCollection;
-use Algatux\InfluxDbBundle\Services\Clients\Contracts\ClientInterface;
-
 /**
- * Class HttpEvent.
+ * Class HttpEventAbstract.
  */
-class HttpEvent extends InfluxDbEvent
-{
-    public function __construct(PointsCollection $collection)
-    {
-        parent::__construct($collection, ClientInterface::HTTP_CLIENT);
-    }
-}
+final class HttpEventAbstract extends AbstractInfluxDbEvent
```

Comment:
Why `HttpEventAbstract`? This could confuse people thinking it's an abstract class.